packages/portalnetwork/src/networks/history/util.ts

```diff
@@ -41,42 +42,74 @@ import type {
 import type { WithdrawalBytes } from '@ethereumjs/util'
 import type { ForkConfig } from '@lodestar/config'
 
+export const BlockHeaderByNumberKey = (blockNumber: bigint) => {
+  return Uint8Array.from([0x03, ...BlockNumberKey.serialize({ blockNumber })])
```

Comment:
Is `0x03` the ContentType?  If so, can we replace `0x03` with `HistoryNetworkContentType.BlockHeaderByNumber`?

Reply:
yes